Resources/Maps/atlas.yml

```diff
@@ -4194,6 +4120,10 @@ entities:
           showEnts: False
           occludes: True
           ent: null
+        fuelSlot: !type:ContainerSlot
+          showEnts: False
+          occludes: True
+          ent: null
```

Comment:
Are these containers supposed to be here.

Reply:
unless it is overriding default slot no